src/plugins/paginate/_base.scss

```diff
@@ -24,6 +24,67 @@
 		border-bottom-right-radius: 4px;
 		border-top-right-radius: 4px;
 	}
+
+	li.active:nth-last-child(2) button {
+		border-bottom-right-radius: 4px;
+		border-top-right-radius: 4px;
+	}
+
+	.pagination > li > button, .pagination > li > span {
+		background-color: #eaebed;
+		border: 1px solid #dcdee1;
+		color: #335075;
+		float: left;
+		line-height: 1.4375;
+		margin-left: -1px;
+		position: relative;
+		text-decoration: none;
+	}
+
+	.pagination > .active > button, .pagination > .active > button:focus, .pagination > .active > button:hover, .pagination > .active > span, .pagination > .active > span:focus, .pagination > .active > span:hover {
```

Comment:
Please remove mentions of `> span` as it is not element that is added in the UI.

Reply:
Fixed in commit "Made recommended changes".